frontend/pages/admin/IntegrationsPage/cards/Calendars/_styles.scss

```diff
@@ -30,6 +34,7 @@
 
   #oauth-scopes {
     font-family: "SourceCodePro", $monospace;
+    color: $core-fleet-black;
```

Comment:
@jacobshandling it's black.

Maybe the issue on your PR of not seeing updated styling is happening here too?

Reply:
Weird, yea now I'm seeing it as black.